tooling/v8-snapshot/src/generator/blueprint.ts

```diff
@@ -109,101 +109,110 @@ export function scriptFromBlueprint (config: BlueprintConfig): {
 
```

Comment:
This file is more easily reviewed by ignoring whitespace changes.

<img width="497" alt="image" src="https://user-images.githubusercontent.com/4873279/206357264-7f7e9440-0419-49f1-b19e-310d4b380f27.png">